packages/grails.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Install a custom version of Grails, https://grails.org
+#
+# Add at least the following environment variables to your project configuration
+# (otherwise the defaults below will be used).
+# * GRAILS_VERSION
+#
+# Include in your builds via
+# source /dev/stdin <<< "$(curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/grails.sh)"
+
+GRAILS_VERSION=${GRAILS_VERSION:="3.2.9"}
+GRAILS_PATH=${GRAILS_PATH:="$HOME/grails"}
+
+CACHED_DOWNLOAD="${HOME}/cache/grails-v${GRAILS_VERSION}.zip"
+
+mkdir -p ${GRAILS_PATH}
+wget --continue --output-document "${CACHED_DOWNLOAD}" "https://github.com/grails/grails-core/releases/download/v${GRAILS_VERSION}/grails-${GRAILS_VERSION}.zip"
+unzip -q -o "${CACHED_DOWNLOAD}" -d "${GRAILS_PATH}"
```

Comment:
It would be great if we can unzip Grails to `GRAILS_PATH` and not a subdirectory. `tar` supports a `--strip-components` option, which is very useful for this, but I'm not sure if `zip` offers something similar.